templates/base.pkr.hcl

```diff
@@ -67,6 +67,17 @@ build {
     ]
   }
 
+  // Add GitHub to known hosts
+  // Similar to https://github.com/actions/runner-images/blob/main/images/macos/scripts/build/configure-ssh.sh
+  provisioner "shell" {
+    inline = [
+      "source ~/.zprofile",
+      "[[ ! -d ~/.ssh ]] && mkdir ~/.ssh 2>/dev/null",
+      "chmod 777 ~/.ssh",
+      "ssh-keyscan -t rsa,ecdsa,ed25519 github.com >> ~/.ssh/known_hosts",
```

Comment:
I don't think this is secure, as there's always possibility of MITM 🤔

How about we add static `~/.ssh/known_hosts` from [GitHub's SSH key fingerprints
](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/githubs-ssh-key-fingerprints) help article instead?